test/unit/model/v2/test_platforms_section.py

```diff
@@ -44,7 +44,7 @@ def _model_platforms_docker_section_data():
                 'url': 'registry.example.com',
                 'credentials': {
                     'username': 'username',
-                    'password': '$PASSWORD',
+                    'password': '$PASS_WORD',
```

Comment:
why?

Reply:
Originally this MR was for allowing shell variables with underscores.  That specific line was updated to represent a variable with an underscore.  Now it's superfluous, but still a valid test example.

Reply:
Ah, alright. So please add this information as a comment at the end of this line.

Reply:
Really? It's fine as is IMHO ...

Reply:
It wasn't obvious to me at all wrt the motivation of this change. Somebody may decide to refactor this and remove that underscore back in future. Most folks wouldn't check the history of this change. And also if they do the PR title contains literally zero clues of what it's about meaning that somebody would probably go further and connect the dots based on 2-3 extra places on the Internet. Now, would you go for all of this hassle if the refactoring reasoning is already obvious to you? (it is usually obvious at the point when you've decided to do it)

This is why I would like to see some clue at least somewhere. Comment should only explain "why", implementation details can be looked up if necessary but it should facilitate making the motivation immediately obvious.

Reply:
I just removed it instead.  It's unnecessary since the scope of this MR changed.